README.rst

```diff
@@ -29,7 +29,13 @@ The basic idea for calculating estimations is the following:
 
 1. `SPRINT_HOURS_RESERVED_FOR_MEETINGS` hours are reserved for the meetings for each sprint.
 2. `SPRINT_HOURS_RESERVED_FOR_EPIC_MANAGEMENT` hours are reserved for epic management for each sprint.
-3. 1 hour is planned for reviewing each task with <= 3 story points. For bigger tasks, 2 hours are reserved.
+3. Reserved time for reviewing is defined in `SPRINT_HOURS_RESERVED_FOR_REVIEW`, where you can define the review time for:
+    * Less than 2 story points("<2"), default is 0.5 hours
+    * 2 story point("2"), default is 1 hour
+    * 3 story points("3"), default is 2 hours
+    * 5 story points("5"), default is 3 hours
+    * More than 5 story points(">5"), defaults to 5 hours
+    * When a task is not estimated("0"), defaults to 2 hours
```

Comment:
I see that it's an empty `dict` by default in the settings. Am I missing something?

Reply:
Changed the default to meet the defined allocation logic in the issue description